components/hab/src/command/studio/docker.rs

```diff
@@ -183,13 +183,14 @@ where
     S: AsRef<OsStr>,
     T: AsRef<str>,
 {
-    let mut cmd_args: Vec<OsString> = vec![
-        "run".into(),
-        "--rm".into(),
-        "--tty".into(),
-        "--interactive".into(),
-        "--privileged".into(),
-    ];
+    let mut cmd_args: Vec<OsString> = vec!["run".into(), "--rm".into(), "--privileged".into()];
+    match args.first().map(|f| f.to_str().unwrap_or_default()) {
+        Some("build") => {}
```

Comment:
@reset You know what I was after, right? Just could not find a much more elegant way to describe this. Seemed like `if let …` doesn't like `!=` bools.

Reply:
Yeah this was perfectly fine